addons/mail/static/src/components/discuss/discuss.js

```diff
@@ -10,25 +11,15 @@ export class Discuss extends LegacyComponent {
      * @override
      */
     setup() {
-        this._updateLocalStoreProps();
         // bind since passed as props
         useUpdate({ func: () => this._update() });
+        useUpdateToModel({ methodName: 'onComponentUpdate' });
     }
 
     _update() {
         if (this.discussView.discuss.thread) {
             this.trigger('o-push-state-action-manager');
         }
```

Comment:
Note that this could also be moved in models: by using `useComponentToModel` in this component, and do `this.component.trigger()` in `DiscussView/onComponentUpdate`.